docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,338 @@
+  8. Click Edit to create your application.
+
+#### Step 2: Install required packages:
+  [PUT](https://docs.snowflake.com/en/sql-reference/sql/put) the [environment.yml](/environment.yml) file to the stage location specified by the `STREAMLIT_ROOT_LOCATION` parameter of the Streamlit object:
+  ```
+  $ SELECT STREAMLIT_ROOT_LOCATION FROM <database>.INFORMATION_SCHEMA.STREAMLITS;
+
+  > <database>.<schema>."<appname> (Stage)"
+  ```
+  ```
+  PUT file:///<full_path>/environment.yml '@<database>.<schema>."<appname> (Stage)"' AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
+  ```
+
+#### Step 4: Provide the geojson [file](/countries.geo.json) that contains the country information required to render the choropleth map:
+  ```
+  PUT file:///<full_path>/countries.geo.json '@<database>.<schema>."<appname> (Stage)"' AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
+  ```
+
+#### Step 5: Copy the code from [snowplow.py](/snowplow.py) inside the editor
+
+and Click `Run`
+
+
+### 3. Run it on Streamlit's cloud
+
+Follow the instruction on [https://streamlit.io/cloud](https://streamlit.io/cloud)
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+</TabItem>
+
+</Tabs>
+
+## Dashboards
+
+### Acquisition Overview
+
+The **Acquisition Overview** aims at updating you about the website traffic with a breakdown by marketing channels.
+
+There is also a distinction between `New Users` who have not previously had a session on the website (domain_sessionidx=1) and `Existing users` who have a session with domain_sessionidx > 1.
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_AO.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_AO.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_AO.png)
+
+</TabItem>
+
+</Tabs>
+
+### User Acquisition
+This dashboard is about **User Acquisition** which is effectively about what channels / methods were used to acquire new users.
+
+Every visual is already filtered on new users only (domain_sessionidx = 1).
+
+Metrics definitions:
+
+**engaged_sessions** = count(case when engaged=True end)
+**engagement_rate** = engaged_sessions / count(distinct domain_sessionid)
+**conversion_count** =sum(cv__all_volume)
+**conversion_revenue** = sum(cv__all_total)
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_UA.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_UA.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_UA.png)
+
+</TabItem>
+
+</Tabs>
+
+### Traffic Acquisition
+
+This dashboard is about **Traffic Acquisition** which aims at highlighting which channels are used to re-engage existing (returning) users.
+
```

Comment:
Terminology wise, are these different dashboards, or pages in the same dashboard?

Reply:
"and what each visualization view (= page / tab / dashboard depending on the tool) " - explain it at the beginning and call them view/page later on. I'm sure they will understand